core/src/data/ser/debug.cpp

```diff
@@ -136,6 +138,13 @@ bool DebugSerializer::decompose(const Type& type, const void* value)
         this->separate(true);
         mStream.put(']');
     }
+    else if (type.has<EnumTrait>())
+    {
+        const auto& trait = type.get<EnumTrait>();
+        mStream.put('"');
+        mStream.print(trait.variant(value).name());
+        mStream.put('"');
```

Comment:
```suggestion
        mStream.print(trait.variant(value).name());
```
An enum variant is more of a type than a string, I would remove the quotes.